src/matchers/font.rs

```diff
@@ -1,27 +1,19 @@
 /// Returns whether a buffer is WOFF font data.
 pub fn is_woff(buf: &[u8]) -> bool {
-    buf.len() > 7
+    buf.len() > 3
         && buf[0] == 0x77
         && buf[1] == 0x4F
         && buf[2] == 0x46
         && buf[3] == 0x46
-        && buf[4] == 0x00
-        && buf[5] == 0x01
-        && buf[6] == 0x00
-        && buf[7] == 0x00
 }
 
 /// Returns whether a buffer is WOFF2 font data.
 pub fn is_woff2(buf: &[u8]) -> bool {
-    buf.len() > 7
+    buf.len() > 3
         && buf[0] == 0x77
         && buf[1] == 0x4F
         && buf[2] == 0x46
         && buf[3] == 0x32
-        && buf[4] == 0x00
-        && buf[5] == 0x01
-        && buf[6] == 0x00
-        && buf[7] == 0x00
```

Comment:
Do we just need to keep the last four bytes?

See https://github.com/sindresorhus/file-type/blob/99cb0198bf3e55cfeda0c407607dddbab8875d91/core.js#L529-L553